### ISSUE TITLE
Mixed mutators replay error

### ISSUE THREAD

**Issue by Buszmen** [tfausak/octane#73]
I was trying to figure out all mutators and I created a monster replay and mixed all mutators I could. As a result I get an error:
```
octane.exe: Data.Binary.Get.runGet at position 553: don't know how to get attribute value Text {textValue = "TAGame.VehiclePickup_TA:bNoPickup"}
CallStack (from HasCallStack):
  error, called at libraries\binary\src\Data\Binary\Get.hs:342:5 in binary-0.8.3.0:Data.Binary.Get
```
Windows build 0.18.3
Same for linux build 0.18.2.

[Link to replay](https://www.dropbox.com/s/idhhstxsxktkh6i/520E1BFF468CF6C3C48D1EA85D9C7909.replay?dl=0)

**Comment by tfausak** [comment]
Thanks for reporting this! The unknown attribute (`TAGame.VehiclePickup_TA:bNoPickup`) starts with a "b", which means its a boolean attribute. Hopefully this will be an easy fix.

**Comment by tfausak** [comment]
Fixed by 8ebf7ed0252e8cc2e662ff2813af01a8160ade10 and released as [version 0.19.0](https://github.com/tfausak/octane/releases/tag/0.19.0), which should be available just as soon as the build finishes.
